README.md

```diff
@@ -936,6 +956,43 @@ const MyComponent = () => (
 )
 ```
 
+### 7.7 Avoid compare directly strings
+
+When know all possible values we can use enum to achieve better readability, control.
```

Comment:
```suggestion
When know all possible values we can use enum to achieve better readability and control.
```

Reply:
thanks!! fix in [91eaf3b](https://github.com/juntossomosmais/frontend-guideline/pull/29/commits/91eaf3b5bc124ec841b837a8113ac43c04012258)